### PR TITLE
fix(API): ISSUE-185 Support paging when receiving tags

### PR DESCRIPTION
**Pull request by ivanilves**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -32,7 +32,6 @@ script:
   - if [[ "${TRAVIS_PULL_REQUEST}" != "false" ]]; then make semantic RANGE=$(scripts/travis_range.sh); fi
   - make coverage
   - make blackbox-integration-test
-  - make docker-image DOCKER_TAG=release
```
Comment (ivanilves) [inline .travis.yml]: https://github.com/ivanilves/lstags/issues/191 😭

Comment (vonrabbe) [reply .travis.yml]: agree. unfortunately travis went south 📉

File: api/v1/registry/client/request/request.go [hunk 2]
```diff
@@ -109,5 +109,18 @@ func Perform(url, auth, mode string, trace bool, retries int, delay time.Duratio
...
-	return resp, err
+	return resp, getNextLink(resp.Header["Link"]), err
+}
+
+func getNextLink(headers []string) string {
+	if len(headers) == 0 {
+		return ""
+	}
+
+	nextlink := headers[0]
```
Comment (vonrabbe) [inline api/v1/registry/client/request/request.go]: only first element is important? 🤔

Comment (ivanilves) [reply api/v1/registry/client/request/request.go]: YES!

**Pull request merged**